front/src/modules/simulationResult/components/ManchetteWithSpaceTimeChart/ManchetteWithSpaceTimeChart.tsx

```diff
@@ -145,6 +145,13 @@ const ManchetteWithSpaceTimeChartWrapper = ({
     initialDepartureTime: Date;
   }>();
 
+  const isTimetableItemValid = useMemo(() => {
+    const timetableItemUsedForProjectionWithDetails = timetableItemsWithDetails?.find(
+      (item) => item.id === selectedProjectionId
+    );
+    return timetableItemUsedForProjectionWithDetails?.summary?.isValid ?? true;
```

Comment:
Maybe it should be false by default no ? 
```suggestion
    return timetableItemUsedForProjectionWithDetails?.summary?.isValid ?? false;
```

Reply:
Done